src/components/ImageUpload/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+interface ImageUploadProps {
+  onImageSelect: (imageSrc: string) => void;
+}
+
+const ImageUpload: React.FC<ImageUploadProps> = ({ onImageSelect }) => {
```

Comment:
```suggestion
const ImageUpload = ({ onImageSelect }: ImageUploadProps) => {

```
React.FC 사용을 지양해주세요.
그리고 이 컴포넌트는 시연영상에서 보이지 않는 것 같은데 어디서 사용되는 것인가요?